pkg/pxc/backup/job.go

```diff
@@ -46,6 +46,11 @@ func (bcp *Backup) JobSpec(spec api.PXCBackupSpec, cluster api.PerconaXtraDBClus
 
 	manualSelector := true
 	backbackoffLimit := int32(10)
+	verifyTLS := "1"
+	if cluster.Backup.Storages[spec.StorageName].VerifyTLS != nil &&
+		!*cluster.Backup.Storages[spec.StorageName].VerifyTLS {
+		verifyTLS = "0"
+	}
```

Comment:
can we keep it simple?
```suggestion
	verifyTLS := true
	if cluster.Backup.Storages[spec.StorageName].VerifyTLS != nil {
		verifyTLS = *cluster.Backup.Storages[spec.StorageName].VerifyTLS
	}
```

Reply:
Resolved in 9fbab09b01d7c2b1b6b840fe02ce350b06633618